index.js

```diff
@@ -323,7 +334,7 @@ module.exports = function (config) {
     return rootStep;
   }
 
-  // Checks if step is actually BDD
+  // Checks if step is actually BDDconfig.includeExampleValues
```

Comment:
looks like you accidentally updated this?

Reply:
ugh...sorry man.  just sloppy 😞 .  fixed now

Reply:
no worries